src/color/setting.js

```diff
@@ -340,7 +340,7 @@ p5.prototype.colorMode = function(mode, max1, max2, max3, maxA) {
 
 /**
  * Sets the color used to fill shapes. For example, if you run
- * fill(204, 102, 0), all subsequent shapes will be filled with orange. This
+ * fill(204, 102, 0), all shapes run subsequent to the fill command will be filled with the color orange. This
```

Comment:
```suggestion
 * fill(204, 102, 0), all shapes drawn after the fill command will be filled with the color orange. This
```

For further clarity.

Reply:
i agree; "drawn after" definitely provides further clarity than "subsequent", thank you @limzykenneth